dwave/embedding/transforms.py

```diff
@@ -548,7 +548,7 @@ def unembed_sampleset(target_sampleset, embedding, source_bqm,
             already exists in the sample set then it is overwritten.
 
     Returns:
-        :obj:`.SampleSet`: Sample set in the source BQM.
+        :obj:`~dimod.SampleSet`: Sample set in the source BQM.
```

Comment:
Using `:obj:` for classes ([instead of `:class:`](https://www.sphinx-doc.org/en/master/usage/domains/python.html#cross-referencing-python-objects)) remains such an eyesore to me, to this day. :shrug: